modules/va_forms/app/sidekiq/va_forms/form_reloader.rb

```diff
@@ -6,17 +6,60 @@ module VAForms
   class FormReloader
     include Sidekiq::Job
 
-    sidekiq_options retries: 7
+    sidekiq_options retry: 7
 
-    def perform
-      Rails.logger.info("#{self.class.name} is being called.")
+    STATSD_KEY_PREFIX = 'api.va_forms.form_reloader'
+
+    sidekiq_retries_exhausted do |msg, _ex|
+      job_id = msg['jid']
+      error_class = msg['error_class']
+      error_message = msg['error_message']
+
+      StatsD.increment("#{STATSD_KEY_PREFIX}.exhausted")
+
+      message = 'VAForms::FormReloader retries exhausted'
+      Rails.logger.error(
+        message,
+        { job_id:, error_class:, error_message: }
+      )
+      VAForms::Slack::Messenger.new(
+        {
+          class: 'VAForms::FormReloader',
+          exception: error_class,
+          exception_message: error_message,
+          detail: message
+        }
+      ).notify!
```

Comment:
Another option would have been to trigger the Slack notification from Datadog when the `StatsD` metric was incremented or when the error was logged. However, there isn't a connection yet between the VA.gov Datadog instance and the Lighthouse Slack, so I opted to trigger the alert from the job itself until that connection has been made. (The connection request is in progress.)